Xwt.Mac/Xwt.Mac/ContextBackendHandler.cs

```diff
@@ -276,7 +276,8 @@ public override void DrawTextLayout (object backend, TextLayout layout, double x
 		{
 			CGContext ctx = ((CGContextBackend)backend).Context;
 			SetupContextForDrawing (ctx);
-			MacTextLayoutBackendHandler.Draw (ctx, Toolkit.GetBackend (layout), x, y);
+			var li = ApplicationContext.Toolkit.GetSafeBackend (layout);
```

Comment:
This should be handled by the frontend. DrawTextLayout should take a TextLayout backend (object) instead of directly a TextLayout, so the caller will do the GetSafeBackend() call and there will be no need for platform-specific checks.